lib/config/config-initializer.js

```diff
@@ -306,6 +307,15 @@ function promptUser(callback) {
             }
         },
         {
+            type: "confirm",
+            name: "airbnbReact",
+            message: "Do you use React?",
```

Comment:
It shouldn't block this PR, but I noticed that we have a prompt for the same question [later in the file](https://github.com/eslint/eslint/blob/e5a9a1276036b916b32cd639435cbecbc67734e0/lib/config/config-initializer.js#L402-L410). It would be nice if we could combine those somehow to avoid code duplication.